src/scripts/admin/filters/button-enhanced.js

```diff
@@ -0,0 +1,144 @@
+/* eslint-disable react/jsx-props-no-spreading, react/prop-types */
+import classnames from 'classnames';
+
+const { assign, merge } = require('lodash');
+
+const { __ } = wp.i18n;
+const { addFilter } = wp.hooks;
+const { createHigherOrderComponent } = wp.compose;
+const { Fragment } = wp.element;
+const { InspectorControls } = wp.blockEditor;
+const { PanelBody, SelectControl } = wp.components;
+
+wp.blocks.registerBlockVariation('core/buttons', {
+	name: 'bcgov-buttons',
```

Comment:
Im curious of why a plural and a single version, this might be a good use of comments, otherwise future self, and every one else will be asking... why did we do this.

Reply:
Good point. Missed adding a fulsome comment for this and will remedy. These two variations (plural and singular) add BCGov versions of the core buttons (group) and button (contained in the group). What I haven't done is determine whether I should attempt to suppress the display of core/buttons and core/button blocks in the general blocks list and simply offer these variations, or just add default styling for government styling to the core versions themselves. Will be having that discussion with the UX designer this morning.